index.html

```diff
@@ -0,0 +1,315 @@
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <footer>
+      <form action="https://formspree.io/f/mgernvdd" method="POST" class="f-box-wrapper">
+        <h2 class="contact-h2">Contact Me</h2>
+        <p class="contact-me-text">
+          If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding. 
+          I’d love to help with it
+        </p>
+
+          <input type="text" name="name" maxlength="30" placeholder="Enter your Full Name" required />
+          <input type="email" name="_replyto" placeholder="Enter your email" required/>
+          <textarea name="message" maxlength="500" placeholder="Type your message here" required></textarea>
+        
+          <button type="submit" class="submit">Get In Touch</button>
```

Comment:
I think centering the button text would make the button look better and more like the design spec. Additionally, please implement button interactions such as hover, active (when it's pressed), disabled, and enabled (default state) as shown in the Figma template.